catalog/app/website/pages/AwsMarketplace/CaseStudies.tsx

```diff
@@ -0,0 +1,154 @@
+import cx from 'classnames'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+import logoCelsius from './logo-celsius.png'
+import logoResilience from './logo-resilience.png'
```

Comment:
New Resilience Logo: 
![Resilience-1](https://user-images.githubusercontent.com/116831980/213199022-63e97bc2-f687-44bd-a6b1-6a61626c5700.jpeg)

Reply:
I'll use this, if you don't mind https://resilience.com/wp-content/uploads/2021/11/Resilience-Logo.svg